### PR TITLE
Add Postman.app v3.2.9.1

### PR DESCRIPTION
**Pull request by knilob**
Created a Cask for the Postman Mac app currently in Beta.
Postman is an API client used for testing API endpoints.

### CODE REVIEW

File: Casks/postman.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'postman' do
+  version '3.2.9.1'
+  sha256 '1e62914e81fdb9ea5e00520777cee48570182a77a6771be5f63ca1681454cac4'
+
+  url "https://s3.amazonaws.com/postman-electron-builds/mac/postman-osx-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/postman.rb]: Needs a `# amazonaws.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by knilob** [comment]
Sorry about that. Should be fixed up now.

**Comment by vitorgalvao** [comment]
Thank you. As soon as Travis passes, we can merge.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 2cb11be4541c2cf49249029cce136a76f90b0698 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**